packages/components/organisms/f-content-cards/stories/ContentCards.stories.js

```diff
@@ -162,58 +225,127 @@ export function ContentCardsCustom (args, { argTypes }) {
+                        },
+                        title: 'Surprise!',
+                        type: 'Post_Order_Card_1',
+                        url: null
+                    });
                 }
-            },
-            customCardsCallback (callback) {
-                this.applyCustomCards = callback;
             }
         },
 
         /**
          * Ensures that card mocks are set up
          */
-        beforeCreate () {
-            resetBrazeData();
-            mock.teardown();
-            mock.setup();
-            mock.post(/\/api\/v3\/content_cards\/sync\/?/, {
-                status: 201,
-                body: JSON.stringify({ cards: [] })
-            });
-            mock.post(/\/api\/v3\/data\/?/, {
-                status: 201,
-                body: JSON.stringify(data())
+        beforeCreate: zeroCardsMockSetup,
+
+        template
+    };
+}
+
+ContentCardsCustom.storyName = 'Custom card';
+
+ContentCardsCustom.args = {
+    apiKey: '00000000-0000-0000-0000-000000000000',
+    userId: 'test-user-id',
+    locale: 'en-GB'
+};
+
+export function ContentCardsLoading (args, { argTypes }) {
+    return {
+        components: {
+            ContentCards
+        },
+
+        props: Object.keys(argTypes),
+
+        methods,
+
+        /**
+         * Ensures that card mocks are set up
+         */
+        beforeCreate: notReturningMockSetup,
+
+        /**
+         * Reject the promise set up by the above mock
+         */
+        beforeDestroy () {
+            this.rejectMockPromise({
+                status: 500
             });
-            mock.use(proxy);
         },
 
         template
     };
 }
 
-ContentCardsCustom.storyName = 'custom card';
+ContentCardsLoading.storyName = 'Loading state';
 
-ContentCardsCustom.args = {
+ContentCardsLoading.args = {
     apiKey: '00000000-0000-0000-0000-000000000000',
     userId: 'test-user-id',
-    title: 'Promotional Offers',
-    locale: 'en-GB',
-    enabledCardTypes: defaultEnabledCardTypes
+    locale: 'en-GB'
+};
+
+export function ContentCardsNoCards (args, { argTypes }) {
+    return {
+        components: {
+            ContentCards
+        },
+
+        props: Object.keys(argTypes),
+
+        methods,
+
+        /**
+         * Ensures that card mocks are set up
+         */
+        beforeCreate: zeroCardsMockSetup,
+
+        template
+    };
+}
+
+ContentCardsNoCards.storyName = 'No Cards state';
+
+ContentCardsNoCards.args = {
+    apiKey: '00000000-0000-0000-0000-000000000000',
+    userId: 'test-user-id',
+    locale: 'en-GB'
+};
+
+export function ContentCardsError (args, { argTypes }) {
+    return {
+        components: {
+            ContentCards
+        },
+
+        props: Object.keys(argTypes),
+
+        methods,
+
+        /**
+         * Ensures that card mocks are set up
+         */
+        beforeCreate: zeroCardsMockSetup,
+
+        template
+    };
+}
+
+ContentCardsError.storyName = 'Error state';
+
+ContentCardsError.args = {
```

Comment:
Not sure what best practice is in storybook, but would be good to split this file up as it's getting pretty big now.

Perhaps having different files imported for each sub-story?